### ISSUE TITLE
Recent cleaning of penile foreskin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10016]
**Preferred term label:**
Recent cleaning of penile foreskin

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Paraphimosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am wondering if it would be better to title the term
Recent history of manipulation of penile foreskin
see https://www.aafp.org/pubs/afp/issues/2000/1215/p2623.html

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK459233/ "Paraphimosis commonly occurs iatrogenically, when the foreskin is retracted for cleaning, placement of a urinary catheter, a procedure such as a cystoscopy, or for penile examination.  Failure to return the retracted foreskin over the glans promptly after the initial retraction can lead to paraphimosis"

So a better formulation might be "prolonged retraction".

**Comment by pnrobinson** [comment]
@MickeySegal 

added

HP:6000935	
Prolonged retraction of penile foreskin	
A past medical history of recently having retracted the penile foreskin for a prolonged period of time. Paraphimosis commonly occurs iatrogenically, when the foreskin is retracted for cleaning, placement of a urinary catheter, a procedure such as a cystoscopy, or for penile examination. Failure to return the retracted foreskin over the glans promptly after the initial retraction can lead to paraphimosis.
